src/v2/layouts/MemberRegistrationCorrection/MemberRegistrationCorrection.tsx

```diff
@@ -0,0 +1,83 @@
+import { Formik } from 'formik';
+import { Header } from '../../components/header';
+import styled from 'styled-components';
+import { MemberCorrectionFormWithAdmin } from './MemberCorrectionFormWithAdmin';
+import { useAPI } from '../../components/hooks';
+import {
+	memberRegistrationCorrectionGETAPI,
+	memberRegistrationCorrectionPUTAPI,
+} from './MemberRegistrationCorrectionAPI';
+import {
+	editStudentData,
+	getStudentData,
+	MemberCorrectionSchemaWithAdmin,
+} from './memberCorrectionSchemaWithAdmin';
+import { useContext, useState } from 'react';
+import { StringsContext } from '../../components/providers/StringsProvider/StringsContext';
+import { MemberCorrectionResponseModal } from './MemberCorrectionResponseModal';
+
+export const MemberRegistrationCorrection = (): JSX.Element => {
+	const { strings } = useContext(StringsContext);
+	const { response } = useAPI(memberRegistrationCorrectionGETAPI({ id: 1, secretKey: '451725' }));
```

Comment:
Remove hardcoding. Create a context say MemberContext and get member id and secretKey from there.
```suggestion
	const { response } = useAPI(memberRegistrationCorrectionGETAPI({ id: <id from context>, secretKey: <secretKey from context> }));
```